src/PlusDataCollection/WinProbe/vtkPlusWinProbeVideoSource.cxx

```diff
@@ -711,6 +770,32 @@ PlusStatus vtkPlusWinProbeVideoSource::InternalConnect()
   {
     SetPWIsEnabled(true);
   }
+  if(m_Mode == Mode::ARFI)
+  {
+    SetARFIIsEnabled(true);
```

Comment:
This whole block of code is a duplicate of, and should be replaced by `this->SetARFIEnabled(true);`